packages/core/scss/components/_radiobuttons.scss

```diff
@@ -33,8 +33,8 @@
 
   [type='radio'] + label:before {
     content: '';
-    width: $size;
-    height: $size;
+    width: 20px;
```

Comment:
@goncalosard Look at line :11 there is already a $size variable which is reused by other classes like `chechmark` etc.